.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ parameters:
     # 3. Commit this change to the PR branch where the changes exist.
     current_golden_images_hash:
         type: string
-        default: a095ed1ef88686fdb24936dad763598ac7981193
+        default: 93ad92a2fc031db0277fee635434955e438e94a9
```

Comment:
VRTs showed a fix in the checkbox which is now picking up the correct corner radius for Spectrum 2 🥳